docs/for-ops/console/usermgnt.md

```diff
@@ -0,0 +1,79 @@
+---
+slug: user-management
+title: Platform - User Management
+sidebar_label: User Management
+---
+
+:::info
+User Management will only be available when Keycloak is configured as the Identity Provider (default). When OIDC is configured, the User Management section will not be available.
+:::
+
+:::info
+User Management can only be used to manage the users of the Platform instance (single cluster).
```

Comment:
Are these features mutually exclusive? 
What to do with bootstrapping the platform-admin user in case of OIDC is enabled?

Reply:
Good question, but not part of this scope. This needs to be addressed in the changes related to APL-251